src/app/components/sandbox-controller/sandbox-controller.component.ts

```diff
@@ -66,19 +69,21 @@ export class SandboxControllerComponent implements OnInit {
     protected _executorSvc: ExecutorService,
     protected _streamBuilder: StreamBuilderService,
     protected _runtimeSvc: RuntimeService,
+    protected _hashedExampleSvc: HashedExampleService,
     protected _logger: LoggerService,
   ) {
     this._maxSources = maxSources ?? this._maxSources;
   }
 
   protected renderExample(): void {
     const examples = this._examples.reduce(
-      (p, c) => ({ ...p, [c.name]: c }), { [this._startExample.name]: this._startExample }
+      (p, c) => ({ ...p, [c.name]: c }), { [this._startExample.name]: this._startExample },
     );
 
     const exampleToRender$ = this._route.paramMap.pipe(
-      map((params) => params.get('exampleName') ?? this._startExample.name),
-      map((name) => examples[name]),
+      map((params) => params.get(RouteParamKeys.ExampleName) ?? this._startExample.name),
+      mergeMap((name) => name !== RouteNames.SharedExample ? of(examples[name]) : this._hashedExampleSvc.getExample(this._route)),
+      map((example) => example ?? this._startExample),
```

Comment:
When going to a shared example the resources is not hidden for empty links:
![image](https://user-images.githubusercontent.com/1577821/225537325-5d82bf35-8d78-4985-9470-2a82bc654572.png)